src/sub.fs

```diff
@@ -0,0 +1,90 @@
+namespace Elmish
+
+open System
+
+/// SubId - Subscription ID, alias for string list
+type SubId = string list
+
+/// Subscription - Generates new messages when running
+type Sub<'msg> =
+    { SubId: SubId
+      Start: SubId -> Dispatch<'msg> -> IDisposable }
```

Comment:
Why do we need to pass SubId into Start?

Reply:
Seemed like a good idea at the time, but now I’m struggling to come up with an example where I would use it. I’ll take it out when I get back to a computer, Mon.

Reply:
Removed